core/src/relabeling/LLRelabelingStrategy.cpp

```diff
@@ -0,0 +1,95 @@
+/*-------------------------------------------------------------------------------
+  This file is part of generalized random forest (grf).
+
+  grf is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  grf is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with grf. If not, see <http://www.gnu.org/licenses/>.
+ #-------------------------------------------------------------------------------*/
+
+#include "relabeling/LLRelabelingStrategy.h"
+
+namespace grf {
+
+LLRelabelingStrategy::LLRelabelingStrategy(double split_lambda,
```

Comment:
I assume we're planning to introduce a causal version? In that case it'd be best to call this `LLRegressionRelabelingStrategy`. That would match naming elsewhere, like `LLCausalPredictionStrategy`.